### DISCUSSION TITLE
Can i get Data from another Budibase App Database?

### DISCUSSION THREAD

**Discussion by steffen-mzk** [Budibase/budibase#4424] [Help] [answered]
Hey there, 
I have two Budibase Apps. In one app i have a budibase database with images. 
now i want to have access to that images from the other app in budibase. 
Like a media library.

Or is there a way to have global Databases to give apps access to it? 

like the user table in budibase. This is in my point of view exactly a global table.

**Comment by mjashanks** [comment, marked as answer]
We will be adding the concept of Global Datasources, probably in the second half of 2022. However, this will be a paid (not open source) feature.

At the end of this month, we will be releasing the Budibase REST API. So, you could use this to access data from another app. I.e. set up a REST API connection from one app to another.
